### PR TITLE
Fix login and register form

### LINKED ISSUE #159: Fix login and register form
Make them forms and and enable auto-complete

### PR DESCRIPTION
**Pull request by afonsofsdomingues**
closes #159

### CODE REVIEW

**Review by JeronimoMendes — CHANGES_REQUESTED** [review]
Delete `data/` and add it to `.gitignore`

File: frontend/.gitignore [hunk 1]
```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/data
```
Comment (JeronimoMendes) [inline frontend/.gitignore:25]: This should be in the root `.gitignore`

File: frontend/src/components/register/RegisterInput.js [hunk 2]
```diff
@@ -110,6 +110,12 @@ const RegisterInput = () => {
 		}
 	};
 
+	const handleKeyPress = (e) => {
+		if (e.keyCode === 13) {
+			handleClick();
+		}
+	};
+
```
Comment (JeronimoMendes) [inline frontend/src/components/register/RegisterInput.js:118]: Isn't there a way to avoid using this?

**Review by JeronimoMendes — CHANGES_REQUESTED** [review]

File: frontend/src/components/register/RegisterInput.js (hunk 2 above)
Comment (afonsofsdomingues) [reply frontend/src/components/register/RegisterInput.js:118]: Not sure, tried multiple approaches, none of them worked.

**Pull request merged**